File: C_by_Harry/CH-4_practice-set/05_problem.c
//todo 5. Write a program to sum first ten natural numbers using while loop.

#include <stdio.h>

int main(){
    int n = 10,s=0;
    for(int i = 1; i<(n+1); i++ ){
        s = s + i;
    }
    printf("the sum of first 10 natural numbers is = %d",s);
    return 0;
}

File: python_by_harry/CH-7 practice set/05_problem.py
'''
5. Write a program to find the sum of first n natural numbers using while loop. 

'''
n = int(input("Enter a number : "))
# s = 0
# i = 0
# while(i<n+1):
#     s += i
#     i+=1

s = 0
for i in range(1,n+1):
    s = s+i



print(f"Sum of natural numbers upto {n} is = {s}")